test/functional/test-fixed-layer.js

```diff
@@ -597,6 +597,7 @@ describe('FixedLayer', () => {
       element1.computedStyle['position'] = 'fixed';
       element1.offsetWidth = 10;
       element1.offsetHeight = 10;
+      element1.computedStyle['top'] = '0px';
```

Comment:
No, not really - `getComputedStyle` always returns resolved pixel values. So, it's best to mimic it the same way in tests.